### PR TITLE
Defect/griz-384

### PR DESCRIPTION
**Pull request by JonathonDonaghy**
final pull request for this batch of bugs. fixes include:

vendor typeahead dropdown now overlays the image in edit product

typeahead will now more reliably show the correct result

added the clear button to the vendor typeahead

fixed the typeahead no results button to clear the current input, rather than set it to a false value (that could be sent to the DB, resulting in an error)

### CODE REVIEW

File: src/actions/productsActions.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import isEmpty from '../validation/is-empty';
 
 // Get Product List
 export const getProducts = index => dispatch => {
+  console.log("in get product");
```
Comment (sophia-cg) [inline src/actions/productsActions.js]: you left a comment

File: src/actions/productsActions.js [hunk 2]
```diff
@@ -423,13 +436,15 @@ export const filterProductsByCategory = inputs => dispatch => {
 };
 
 export const refreshProductData = (data, filtered) => dispatch => {
+  console.log("in ref prod");
```
Comment (sophia-cg) [inline src/actions/productsActions.js]: more debug comments

File: src/actions/productsActions.js [hunk 3]
```diff
@@ -423,13 +436,15 @@ export const filterProductsByCategory = inputs => dispatch => {
...
   if (filtered) {
     dispatch({
       type: types.GET_FILTERED_PRODUCTS,
       payload: data,
       filter: filtered.cur_id
     });
   } else {
+    console.log("getting products");
```
Comment (sophia-cg) [inline src/actions/productsActions.js]: here too

**Review by sophia-cg — CHANGES_REQUESTED** [review]
Just need to remove your debug comments, and it's fine

File: src/actions/productsActions.js (hunk 1 above)
Comment (JonathonDonaghy) [reply src/actions/productsActions.js]: opps, fixing

**Review by sophia-cg — APPROVED** [review]

**Review by JenniPeriasamy — APPROVED** [review]

**Pull request merged**